src/k8s-engine/wsl.ts

```diff
@@ -194,7 +196,7 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     return 'wsl';
   }
 
-  protected writeSetting(changed: RecursivePartial<typeof this.cfg>) {
+  protected writeSetting(changed: Record<string, any>) {
```

Comment:
Any reason you're dropping type checking here?

Reply:
That wasn't supposed to get in. It's because goland's use of eslint isn't handling that construct. I filed an issue with jetbrains, haven't heard from them yet. Probably the IDE is using a default package.json, not sure where it is.